Description: Modify config.in to use vendor zlib instead of the built-in one.
Origin: vendor
Forwarded: not-needed
Author: Damyan Ivanov
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-05-08

--- a/config.in
+++ b/config.in
@@ -16,9 +16,9 @@
 #    Setting the Gzip OS Code
 #
 
-BUILD_ZLIB      = True
-INCLUDE         = ./zlib-src
-LIB             = ./zlib-src
+BUILD_ZLIB      = False
+INCLUDE         = /usr/include
+LIB             = /usr/lib
 
 OLD_ZLIB        = False
 GZIP_OS_CODE    = AUTO_DETECT
